src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -136,7 +137,7 @@ def inbox_search(
 
 
 @region_silo_endpoint
-class OrganizationGroupIndexEndpoint(OrganizationEventsEndpointBase):
+class OrganizationGroupIndexEndpoint(OrganizationEndpoint):
```

Comment:
Does not look like this class needs to extend `OrganizationEventsEndpointBase`
extending `OrganizationEndpoint` instead